### PR TITLE
docs: add vitest intro to typescript/testing

### PR DESCRIPTION
**Pull request by zxch3n**
I'm not sure whether a gif screenshot is too bothering. Let me know if you guys prefer code examples or static screenshots ;)

### CODE REVIEW

**Comment by stech11845** [comment]
How do I look at the rendered docs?

**Review by stech11845 — CHANGES_REQUESTED** [review]
I feel like the other ones should be consistent with what you added to Vitest, a Mocha, Playwright, and Jest intro.

**Review by ProKashif — APPROVED** [review]

File: docs/typescript/typescript-testing.md [hunk 1]
```diff
@@ -8,4 +8,9 @@ Visual Studio Code supports testing TypeScript through it's testing framework. Y
...
 ## Jest
 
-## Vitest
\ No newline at end of file
+## Vitest
+
+
+[Vitest](https://vitest.dev/) is an ESM-first, modern unit-test framework powered by Vite. It works out of the box for TypeScript/Node.js projects. You can learn how to get started and read about its features in [the Vitest guide](https://vitest.dev/guide/). To enhance your VS Code experience, you can install [the official VS Code extension](https://marketplace.visualstudio.com/items?itemName=ZixuanChen.vitest-explorer).
+
+![A screenshot of using Vitest extension inside VS Code](./images/testing/vitest.gif)
```
Comment (cadamini) [inline docs/typescript/typescript-testing.md:16]:
```suggestion

[Vitest](https://vitest.dev/) is a modern Enterprise Service Management (ESM) unit-test framework powered by Vite. Vitest works out of the box for TypeScript or Node.js projects. To learn how to get started and about its features, check out the [Vitest guide](https://vitest.dev/guide/). You can enhance your VS Code experience by installing [the official Vitest VS Code extension](https://marketplace.visualstudio.com/items?itemName=ZixuanChen.vitest-explorer).
```

The screenshot does not add value and is very noisy.

Comment (zxch3n) [reply docs/typescript/typescript-testing.md]: Updated. ESM refers to ES modules

**Review by asus831 — APPROVED** [review]

**Review by cadamini — APPROVED** [review]

**Review by Umbrella-Pad — COMMENTED** [review]
compile

**Review by rondales — APPROVED** [review]

**Review by rondales — APPROVED** [review]

**Review by rondales — APPROVED** [review]

**Review by rondales — APPROVED** [review]

**Review by rondales — APPROVED** [review]

**Review by rondales — APPROVED** [review]

**Review by Lo236816 — APPROVED** [review]

**Review by soft2i — APPROVED** [review]
soft

**Comment by kenshanta** [comment]
Is this PR still valid? 👀

**Comment by ntrogh** [comment]
Closing this PR as we need to rewrite the content for testing in TypeScript. Current general testing guidance is available here: https://code.visualstudio.com/docs/editor/testing

**Pull request closed without merging**